server/routes/bookings/bookingsController.ts

```diff
@@ -1,46 +1,40 @@
 import type { RequestHandler } from 'express'
+import { SessionData } from 'express-session'
+import createError from 'http-errors'
 import { VisitService } from '../../services'
 import getPrisonInformation from '../../constants/prisonInformation'
+import { VisitDetails } from '../../services/visitService'
 
 export default class BookingsController {
   public constructor(private readonly visitService: VisitService) {}
 
-  public viewFuture(): RequestHandler {
-    return async (req, res) => {
+  public view(type: SessionData['bookings']['type']): RequestHandler {
```

Comment:
Just one method now to handle future / past / cancelled booking listings, with `switch` to get the appropriate data.